core/src/main/java/com/mapzen/android/graphics/GraphicsModule.java

```diff
@@ -37,11 +35,4 @@
     return new SceneUpdateManager();
   }
 
-  /**
-   * Returns the object used to generate the underlying style string.
-   * @return
-   */
-  @Provides @Singleton public StyleStringGenerator providesStyleStringGenerator() {
```

Comment:
It can't be a singleton because it keeps state related to an individual marker. I could update it to not save state and have it be a singleton again but I think I prefer to leave it as is because I imagine using it to restore markers when a new style is loaded https://github.com/mapzen/android/issues/348